Penteract/Assets/Scripts/AIMovement.cpp

```diff
@@ -95,3 +96,27 @@ bool AIMovement::CharacterInAttackRange(const GameObject* character, const float
 
 	return false;
 }
+
+void AIMovement::SetClipSpeed(UID clipUID, float speed) {
+	if (speed >= 0.f) {
+		ResourceClip* clip = GameplaySystems::GetResource<ResourceClip>(clipUID);
+		clip->speed = speed;
+	}
+}
```

Comment:
Why is this here? It seems either more generic than "movement" or specific to the Melee grunt (if it will only be used by him). So I don't know if it should be here in AIMovement

Reply:
I don't have another generic script for this kind of functions and i think we can change the name of the script because there are a lot of other no-movement functions there.